exercises/nth-prime/runtests.jl

```diff
@@ -0,0 +1,27 @@
+using Test
+
+include("nthprime.jl")
+include("example.jl")
+
+
+@testset "first prime" begin
+    @test main(1) == 2
+end
+
+@testset "no zeroth" begin
+    @test main(0) == "there is no zeroth prime"

```

Comment:
See above, this should throw an error and the test should test if it does so. You can use `@test_throws` to test for errors.